recipes/boost/bld.bat

```diff
@@ -0,0 +1,37 @@
+:: Set the right msvc version according to Python version
+if "%PY_VER%"=="2.7" (
+    set MSVC_VER=9.0
+    set LIB_VER=90
+) else if "%PY_VER%"=="3.4" (
+    set MSVC_VER=10.0
+    set LIB_VER=100
+) else (
+    set MSVC_VER=14.0
+    set LIB_VER=140
+)
+
+:: Start with bootstrap
+call bootstrap.bat
+if errorlevel 1 exit 1
+
+:: Build step
+.\b2 install ^
+    --build-dir=buildboost ^
+    --prefix=%LIBRARY_PREFIX% ^
+    toolset=msvc-%MSVC_VER% ^
+    address-model=%ARCH% ^
+    variant=release ^
+    threading=multi ^
+    link=shared ^
+    -j%CPU_COUNT% ^
+    -s ZLIB_INCLUDE="%LIBRARY_INC%" ^
+    -s ZLIB_LIBPATH="%LIBRARY_LIB%"
+if errorlevel 1 exit 1
+
+:: Install fix-up for a non version-specific boost include
+move %LIBRARY_INC%\boost-1_60\boost %LIBRARY_INC%
```

Comment:
Though it may be more practical for this problem to just use [`PKG_VERSION`]( http://conda.pydata.org/docs/building/environment-vars.html#environment-variables-set-during-the-build-process ).